specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json

```diff
@@ -3879,11 +3879,44 @@
             "Purgeable",
             "Recoverable+Purgeable",
             "Recoverable",
-            "Recoverable+ProtectedSubscription"
+            "Recoverable+ProtectedSubscription",
+            "CustomizedRecoverable+Purgeable",
+            "CustomizedRecoverable",
+            "CustomizedRecoverable+ProtectedSubscription"
           ],
           "x-ms-enum": {
             "name": "DeletionRecoveryLevel",
-            "modelAsString": true
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Purgeable",
+                "description": "It can be purged by a priviliged user after 90 days"
```

Comment:
"privileged" (last "i" -> "e"). If you're using Visual Studio Code, I highly recommend the cSpell and Micorosft Docs extensions (the latter is an extension pack).